src/data/part.lean

```diff
@@ -0,0 +1,387 @@
+/-
+Copyright (c) 2017 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Jeremy Avigad, Simon Hudon
+-/
+import data.equiv.basic
+
+/-!
+# Partial values of a type
+-/
```

Comment:
I think a module doc that is there only to silence the module doc warning is worse than no doc at all, since it won't show up in future documentation todo lists. If you aren't comfortable writing this doc comment, then I would suggest reinstating the nolint exception.

Reply:
I'm planning on making a separate PR for the documentation. This is actually why I originally asked you about splitting that file.